### PR TITLE
Feature/express setup ect

### LINKED ISSUE #2: build server.js file in Express

### LINKED ISSUE #9: Create server.js and add logic for book data GET requests.

### PR DESCRIPTION
**Pull request by ectweitmann**
## What Features are Being Added?
- Created `server.js` and initial logic for handling get requests.

## What is Being Refactored?
- N/A

## Where should the reviewer start?
- `server.js` 

## How should this be manually tested?
- Boot up the server and try a couple of the routes to see if they are working correctly.

## What are the relevant tickets?
- Setting up the initial server.

## Any Other Notes/Questions/Problems?
- Let me know if there are any revisions that need to be made. Thanks team!

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+const express = require('express');
```
Comment (cagallo) [inline server.js:1]: This looks so clean Ethan, we have a functioning local server!

Comment (krdurrett) [inline server.js:1]: Good start to the Express server file

File: server.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+});
+
+app.get('/api/v1/books/:id', (request, response) => {
+  const { id } = request.params;
+  const book = app.locals.books.find(book => {
+    return book.id === id
+  });
+  if (!book) {
+    return response.status(404).json({
+      message: `No novel found with id of #${id}.`
```
Comment (cagallo) [inline server.js:26]: Smart to add this!

File: server.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    });
+  }
```
Comment (krdurrett) [inline server.js:28]: These initial two get functions look perfect!

**Review by cagallo — APPROVED** [review]
Looking great so far! Time to get this data seeded into the db!

**Review by krdurrett — APPROVED** [review]
Great start to the logic using Express to initiate network requests

**Pull request merged**